module/lib/roll.js

```diff
@@ -225,13 +230,14 @@ function buildGearSelectDialog(options, gear) {
 
 function buildTalentSelectDialog(options, talents) {
   console.log("Building Talent Select Dialog", talents, options);
-  let html = "";
-  let selectOptions = "";
-
+  
   if (talents.length === 0) {
-    return;
+    return "";
```

Comment:
It used return undefined that was actually written in the html.

<img width="397" alt="Screenshot 2024-09-22 at 8 12 03 PM" src="https://github.com/user-attachments/assets/a77ff134-e61d-407f-a060-469367c15ef8">